src/plays/password-generator/password-generator-style.css

```diff
@@ -0,0 +1,142 @@
+
+
+.main {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.title {
+    text-align: center;
+}
+
+section {
```

Comment:
Do not override the selectors directly. Always scope it to your play with a class name. The same is applicable to other selectors.